lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -398,6 +398,16 @@ my %stats = (
             };
         },
     },
+    'count.event.eventart.eaa' => {
+        DESC => 'Count of events with artwork',
+        SQL => 'SELECT COUNT(distinct event) FROM event_art_archive.event_art ea
+                  JOIN event_art_archive.event_art_type eat ON ea.id = eat.id
+                WHERE eat.type_id = 1',
+    },
```

Comment:
Right, I'll remove that. I'm also missing these keys from root/statistics/stats.js, so I'll add them there too. Other statistics-related things I haven't done, but maybe can wait until later (?):

 * Adding `count.eventart` as a default line to the timeline (not sure about this yet?).
 * Adding a page like root/statistics/CoverArt.js.

Reply:
We could also rename the page Artwork or something and use it for both, if that seems better.
Similarly, I'd probably just rename "Cover art" to "Art", "Artwork", "images" or whatnot on the timeline and put "Pieces of event art" and "Events with art" under the same header, I think?

Reply:
> We could also rename the page Artwork or something and use it for both, if that seems better.

I renamed the page to "Images" and added two of the event stats to the "Basics" section. I didn't add an "Events" section the the page (because we have nothing else to go there).

> Similarly, I'd probably just rename "Cover art" to "Art", "Artwork", "images" or whatnot on the timeline and put "Pieces of event art" and "Events with art" under the same header, I think?

For this I added a separate "Event art" section, because I believe renaming the category would break existing links.